<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.24">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Data sets used in STRATOS publications – STRATOS TED: Towards Excellent Data</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-a37c72dd2dbac68997fcdc15a3622e78.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-6c6aa755fae961bb06044b69e9d0bf5c.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar docked quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./content.html">Data sets used in STRATOS publications</a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation docked overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
      <a href="./index.html" class="sidebar-logo-link">
      <img src="./Images/stratos_logo.png" alt="" class="sidebar-logo py-0 d-lg-inline d-none">
      </a>
      <div class="sidebar-tools-main">
    <a href="https://github.com/stratosida/" title="" class="quarto-navigation-tool px-1" aria-label=""><i class="bi bi-github"></i></a>
</div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./content.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">Data sets used in STRATOS publications</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./about.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">About</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Data sets used in STRATOS publications</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p>Click on a publication to show details on data access.</p>
<!-- Publication 16 -->
<details>
<summary>
<strong>Heinze G, Baillie M, Lusa L, Sauerbrei W, Schmidt CO, Harrell FE, Huebner M on behalf of TG2 and TG3 of the STRATOS initiative (2024): Regression without regrets –initial data analysis is a prerequisite for multivariable regression. <em>BMC Med Res Methodol 24(178).</em> https://doi.org/10.1186/s12874-024-02294-3 </strong> <br><br> <strong>Keywords:</strong> Initial data analysis, IDA framework, Regression models, Data screening, Reporting, Variable selection, Functional form, Variable transformation, STRATOS Initiative
</summary>
<p><br></p>
<p><a href="https://bmcmedresmethodol.biomedcentral.com/articles/10.1186/s12874-024-02294-3" target="_blank">Publication Link</a> <br><br> <a href="https://zenodo.org/records/7554815" target="_blank">Dataset 1</a> <br> The Bacteremia Dataset. Further information is found in the link.</p>
</details>
<br> <!-- Publication 15 -->
<details>
<summary>
<strong>Boe LA, Shaw PA, Midthune D, Gustafson P, Kipnis V, Park E, Sotres-Alvarez D, Freedman L, on behalf of the Measurement Error and Misclassification Topic Group (TG4) of the STRATOS Initiative (2024): Issues in Implementing Regression Calibration Analyses. <em>American Journal of Epidemiology, 192(8):1406–1414.</em> https://doi.org/10.1093/aje/kwad098 </strong> <br><br> <strong>Keywords:</strong> Berkson error, bias (epidemiology), calibration equation, measurement error, nutritional epidemiology, regression calibration, STRATOS initiative, validation studies
</summary>
<p><br></p>
<p><a href="https://academic.oup.com/aje/article/192/8/1406/7135817?login=true" target="_blank">Publication Link</a> <br><br> “The data used in this paper was obtained through submission and approval of a manuscript proposal to the Hispanic Community Health Study/Study of Latinos Publications Committee, as described on the HCHS/SOL website. For more details, see <a href="https://sites.cscc.unc.edu/hchs/publications-pub" target="_blank">here</a>”</p>
</details>
<br> <!-- Publication 14 -->
<details>
<summary>
<strong>McLernon DJ, Giardiello D, Van Calster B, Wynants L, van Geloven N, van Smeden M, Therneau T, Steyerberg EW, topic groups 6 and 8 of the STRATOS Initiative (2023): Assessing performance and clinical usefulness in prediction models with survival outcomes: practical guidance for Cox proportional hazards models. <em>Annals of Internal Medicine.</em> DOI: https://doi.org/10.7326/M22-0844 </strong> <br><br> <strong>Keywords:</strong> NA
</summary>
<p><br></p>
<p><a href="https://stratos-initiative.org/sites/default/files/2023-07/McLernonetal-2023.pdf" target="_blank">Publication Link</a> <br><br> The breast cancer data for model development from the Netherlands and the breast cancer data for validation from Germany are not publicly available.</p>
</details>
<br> <!-- Publication 13 -->
<details>
<summary>
<strong>Rahnenführer J, De Bin R, Benner A, Ambrogi F, Lusa L, Boulesteix AL, Migliavacca E, Binder H, Michiels S, Sauerbrei W, McShane L, for topic group “High-dimensional data” (TG9) of the STRATOS initiative (2023): Statistical analysis of high-dimensional biomedical data: a gentle introduction to analytical goals, common approaches and challenges. <em>BMC Medicine.</em> DOI: https://doi.org/10.1186/s12916-023-02858-y </strong> <br><br> <strong>Keywords:</strong> High-dimensional data, Omics data, STRATOS intiative, Analytical goals, Initial data analysis, Exploratory data analysis, Clustering, Multiple testing, Prediction
</summary>
<p><br></p>
<p><a href="https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-023-02858-y" target="_blank">Publication Link</a> <br><br> <a href="https://www.bioconductor.org/packages/devel/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf" target="_blank">Dataset 1</a> <br> The Yoruba Dataset. Located on page 60-61 in the link is R code that will load the dataset. <br><br> <a href="https://www.nature.com/articles/nrg2825#Sec4" target="_blank">Dataset 2</a> <br> Data from the 1000 genomes project. This data was used for figure 2. The Link in the publication for “Code and Data” is unfortunately broken. <br><br> <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2164" target="_blank">Dataset 3</a> <br> The GSE2164 Dataset. Data can be downloaded directly at the bottom of website. <br><br> The FLGROSS Dataset is referenced <a href="https://onlinelibrary.wiley.com/doi/10.1002/bimj.201300222" target="_blank">here</a>, but no download is available. <br><br> <a href="https://www.sciencedirect.com/science/article/pii/S0378375807002571?via%3Dihub" target="_blank">Dataset 4</a> <br> The Lymphoma Dataset. Unfortunately the link in section 4.1 is broken. <br><br> <a href="https://montilab.github.io/BS831/articles/docs/quantileNormalization.html" target="_blank">Dataset 5</a> <br> The TCGA HNSCC Subset Dataset. Use the R code in the link to load the data. <br><br> <a href="https://evayiwenwang.github.io/Managing_batch_effects/" target="_blank">Dataset 6</a> <br> The Sponge Metagenomics Dataset. Use the R code in the link to load the data. <br><br> <a href="https://satijalab.org/seurat/archive/v1.4/pbmc3k_tutorial.html" target="_blank">Dataset 7</a> <br> The PBMC Dataset. The raw data is available from a link in the text at the very beginning. <br><br> <a href="https://s3-us-west-2.amazonaws.com/10x.files/samples/cell/pbmc3k/pbmc3k_filtered_gene_bc_matrices.tar.gz" target="_blank">Dataset 8</a> <br> The 10X PBMC Dataset. A direct download link. The analysis referenced in the paper was conducted <a href="https://bioconductor.org/books/3.12/OSCA/clustering.html" target="_blank">here</a>. <br><br> The SSID Project Data is referenced <a href="https://www.degruyter.com/document/doi/10.1515/noise-2020-0011/html" target="_blank">here</a>, but no data is publicly available. <br><br> <a href="https://cran.r-project.org/web/packages/clustra/vignettes/clustra_vignette.html" target="_blank">Dataset 9</a> <br> The Trajectory Data. The link provides code for simulating this dataset from the package clustra. <br><br> <a href="https://bioconductor.org/packages/release/bioc/vignettes/topGO/inst/doc/topGO.pdf" target="_blank">Dataset 10</a> <br> The topGO Dataset. The data is built into the package, and the link goes to a guide on how to prepare and use it. <br><br> <a href="https://www.cancer.gov/ccg/access-data#ccg-genomic-data-resources-by-program" target="_blank">Dataset 11</a> <br> The TCGA Ovarian Cancer Dataset. The Dataset should be available somewhere on the website in the link. <br><br> The CARDIIGAN Dataset is not publicly availabel.The plots are taken from <a href="https://bmjopen.bmj.com/content/7/4/e014467" target="_blank">here</a>.</p>
</details>
<br> <!-- Publication 12 -->
<details>
<summary>
<strong>Little R.J., Carpenter J.R. and Lee K.J. (2022): A Comparison of Three Popular Methods for Handling Missing Data: Complete-Case Analysis, Inverse Probability Weighting, and Multiple Imputation. <em>Sociological Methods &amp; Research</em> </strong> <br><br> <strong>Keywords:</strong> incomplete data, imputation, missing data, weighting
</summary>
<p><br></p>
<p><a href="https://www.stratos-initiative.org/sites/default/files/2022-10/00491241221113873.pdf" target="_blank">Publication Link</a> <br><br> <a href="https://beta.ukdataservice.ac.uk/datacatalogue/studies/study?id=5765" target="_blank">Dataset 1</a> <br> The Youth Cohort Time Series for England, Wales and Scotland, 1984-2002. Needs login to get access to data.</p>
</details>
<br> <!-- Publication 11 -->
<details>
<summary>
<strong>van Geloven N, Giardiello D, Bonneville EF, Teece L, Ramspek CL, van Smeden M, Snell KIE, van Calster B, Pohar-Perme M, Riley RD, Putter H, Steyerberg E, on behalf of the STRATOS initiative (2022): Validation of prediction models in the presence of competing risks: a guide through modern methods. <em>BMJ.</em> DOI: https://www.bmj.com/content/377/bmj-2021-069249 </strong> <br><br> <strong>Keywords:</strong> NA
</summary>
<p><br></p>
<p><a href="https://stratos-initiative.org/sites/default/files/2022-06/bmj-2021-069249.full_.pdf" target="_blank">Publication Link</a> <br><br> <a href="https://github.com/survival-lumc/ValidationCompRisks">Dataset 1</a> <br> The Breast Cancer Dataset.</p>
</details>
<br> <!-- Publication 10 -->
<details>
<summary>
<strong>Lee KJ, Tilling K, Cornish RP, Little RJ, Bell ML, Goetghebeur E, Hogan JW, Carpenter JR for the STRATOS initiative (2021): Framework for the Treatment And Reporting of Missing data in Observational Studies: The TARMOS framework. DOI: https://doi.org/10.1016/j.jclinepi.2021.01.008 </strong> <br><br> <strong>Keywords:</strong> Missing data, Multiple imputation, Observational studies, Reporting, ALSPAC, STRATOS initiative
</summary>
<p><br></p>
<p><a href="https://www.stratos-initiative.org/sites/default/files/2021-03/Lee_2020_new%281%29.pdf" target="_blank">Publication Link</a> <br><br> The ALSPAC data used in the paper is the result of linking multiple publicly available datasets, as described in section 2.</p>
</details>
<br> <!-- Publication 9 -->
<details>
<summary>
<strong>Boulesteix AL, Groenwold R, Abrahamowicz M, Binder H, Briel M, Hornung R, Morris TP, Rahnenführer J, Sauerbrei W for the STRATOS Simulation Panel (2020): Introduction to statistical simulations in health research. DOI: 10.1136/bmjopen-2020-039921 </strong> <br><br> <strong>Keywords:</strong> NA
</summary>
<p><br></p>
<p><a href="https://www.stratos-initiative.org/sites/default/files/2021-09/e039921.full_.pdf" target="_blank">Publication Link</a> <br><br> <a href="https://wwwn.cdc.gov/nchs/nhanes/search/datapage.aspx?Component=Laboratory&amp;CycleBeginYear=2015" target="_blank">Dataset used for Simulation Example</a> <br> “Data from 5092 subjects in the 2015–2016 National Health and Nutrition Examination Survey (NHANES) are used…”, for further information see the chapter “AN EXAMPLE OF A STATISTICAL SIMULATION”.</p>
</details>
<br> <!-- Publication 8 -->
<details>
<summary>
<strong>Andersen PK, Perme MP, van Houwelingen HC, Cook RJ, Joly P, Martinussen T, Taylor JMG, Abrahamowicz M, Therneau TM for the STRATOS TG8 topic group (2020): Analysis of time-to-event for observational studies: Guidance to the use of intensity models. <em>Statistics in Medicine.</em> DOI: 10.1002/sim.8757</strong> <br><br> <strong>Keywords:</strong> censoring, Cox regression model, hazard function, immortal time bias, multistate model, prediction, STRATOS initiative, survival analysis, time-dependent covariates
</summary>
<p><br></p>
<p><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/sim.8757" target="_blank">Publication Link</a> <br><br> <a href="http://publicifsv.sund.ku.dk/~pka/STRATOSTG8/" target="_blank">Dataset 1</a> <br> The PAD Dataset. The link brings you to the author’s website, where you can click on “pad.rda” to download the dataset. <br><br> <a href="https://rochesterproject.org/" target="_blank">Dataset 2</a> <br> The NAFLD Dataset. The link brings you to the website of the Rochester Epidemiology Project. For further information on how the dataset in the paper was generated, see the <a href="https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fsim.8757&amp;file=SIM_8757_intensity_supplement.pdf" target="_blank">Supplement S1</a> from the supporting information in the paper under section 3. <br><br> The Advanced Ovarian Cancer Dataset cannot be found online. The authors of the paper refer to the book “Dynamic Prediction in Clinical Survival Analysis” for further information, and this is a direct citation from this book: “The data originate from two clinical trials comparing different combination chemotherapies that were carried out in The Netherlands around 1980. For details see Neijt et al.&nbsp;(1984) and Neijt et al.&nbsp;(1987).”</p>
</details>
<br> <!-- Publication 7 -->
<details>
<summary>
<strong>Goetghebeur E, le Cessie S, De Stavola B, Moodie E, Waernbaum I on behalf of the topic group Causal Inference (TG7) of the STRATOS initiative (2020): Formulating causal questions and principled statistical answers. <em>Statistics in Medicine.</em> DOI: 10.1002/sim.8757 </strong> <br><br> <strong>Keywords:</strong> causation, instrumental variable, inverse probability weighting, matching, potential outcomes, propensity score
</summary>
<p><br></p>
<p><a href="https://onlinelibrary.wiley.com/doi/10.1002/sim.8741" target="_blank">Publication Link</a> <br><br> The data used in the paper is a simulation based on the “Promotion of Breastfeeding Intervention Trial”. For further information and the necessary code for replication, see Appendix 1/2 under “Supporting Information”.</p>
</details>
<br> <!-- Publication 6 -->
<details>
<summary>
<strong>Keogh RH, Shaw PA, Gustafson P, Carroll RJ, Deffner V, Dodd KW, Küchenhoff H, Tooze JA, Wallace M, Kipnis V, Freedman L (2020): STRATOS guidance document on measurement error and misclassification of variables in observational epidemiology: Part 1 - Basic theory and simple methods of adjustment. <em>Statistics in Medicine.</em> https://doi.org/10.1002/sim.8532 </strong> <br><br> <strong>Keywords:</strong> Berkson error, Classical error, Differential error, Measurement error, Misclassification, Non- differential error, Regression calibration, Sample size, Simulation extrapolation, SIMEX
</summary>
<p><br></p>
<p><a href="https://test1.drupaltest.imbi.uni-freiburg.de/sites/default/files/2021-10/Keoghetal-2020.pdf" target="_blank">Publication Link</a> <br><br> “The OPEN Study data that illustrate the methods presented in this paper are available upon request to RFAB@mail.nih.gov. The request should specify the dataset used in analyses presented in the papers by Keogh et al (2020) and Shaw et al (2020). More information about these data can be obtained at https://epi.grants.cancer.gov/past-initiatives/open/”</p>
</details>
<br> <!-- Publication 5 -->
<details>
<summary>
<strong>Shaw PA, Gustafson P, Carroll RJ, Deffner V, Dodd KW, Keogh RH, Kipnis V, Tooze JA, Wallace M, Küchenhoff H, Freedman L (2020): STRATOS guidance document on measurement error and misclassification of variables in observational epidemiology: Part 2 - More complex methods of adjustment and advanced topics. <em>Statistics in Medicine.</em> https://doi.org/10.1002/sim.8531 </strong> <br><br> <strong>Keywords:</strong> Bayesian methods, Bias analysis, Distribution estimates, Likelihood methods, Moment Reconstruction, Multiple imputation
</summary>
<p><br></p>
<p><a href="https://test1.drupaltest.imbi.uni-freiburg.de/sites/default/files/2021-10/Shawetal-2020.pdf" target="_blank">Publication Link</a> <br><br> “The OPEN Study data that illustrate the methods presented in this paper are available upon request to RFAB@mail.nih.gov. The request should specify the dataset used in analyses presented in the papers by Keogh et al (2020) and Shaw et al (2020). More information about these data can be obtained at https://epi.grants.cancer.gov/past-initiatives/open/”</p>
</details>
<br> <!-- Publication 4 -->
<details>
<summary>
<strong>Huebner M, Vach W, le Cessie S, Schmidt CO, Lusa L on behalf of the Topic Group ‘Initial Data Analysis’ of the STRATOS Initiative (2020): Hidden analyses: a review of reporting practice and recommendations for more transparent reporting of initial data analyses. <em>BMC Medical Research Methodology, 20(1), 1-10.</em></strong> <br><br> <strong>Keywords:</strong> Initial data analysis, Reporting, Observational studies, STRATOS initiative
</summary>
<p><br></p>
<p><a href="https://test1.drupaltest.imbi.uni-freiburg.de/sites/default/files/2021-10/Huebneretal-2020.pdf" target="_blank">Publication Link</a> <br><br> The publication uses data from a literature survey, conducted by the authors.</p>
</details>
<br> <!-- Publication 3 -->
<details>
<summary>
<strong>Wynants L, van Smeden M, McLernon DJ, Timmerman D, Steyerberg EW, Van Calster B on behalf of the Topic Group ‘Evaluating diagnostic tests and prediction models’ of the STRATOS initiative (2019): Three myths about risk thresholds for prediction models. <em>BMC Medicine, 17:192, 1-7.</em></strong> <br><br> <strong>Keywords:</strong> Clinical risk prediction model, Threshold, Decision support techniques, Risk, Data science, Diagnosis, Prognosis
</summary>
<p><br></p>
<p><a href="https://test1.drupaltest.imbi.uni-freiburg.de/sites/default/files/2021-10/Wynants-Three%20myths-BMCmed.pdf" target="_blank">Publication Link</a> <br><br> <a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12916-019-1425-3/MediaObjects/12916_2019_1425_MOESM1_ESM.csv" target="_blank">Dataset 1</a> <br> The Dataset containing only predicted probabilities of malignancy by the ADNEX model and the true outcomes.</p>
</details>
<br> <!-- Publication 2 -->
<details>
<summary>
<strong>Perperoglou A, Sauerbrei W, Abrahamowicz M, Schmid M on behalf of TG2 of the STRATOS initiative (2019): A review of spline function procedures in R. <em>BMC Medical Research Methodology (19:46).</em> DOI: 10.1186/s12874-019-0666-3</strong> <br><br> <strong>Keywords:</strong> Multivariable modelling, Functional form of continuous covariates
</summary>
<p><br></p>
<p><a href="https://test1.drupaltest.imbi.uni-freiburg.de/sites/default/files/2021-10/Perperoglou-Review%20of%20spline%20function%20procedures%20in%20R.pdf" target="_blank">Publication Link</a> <br><br> <a href="https://search.r-project.org/CRAN/refmans/MultiKink/html/triceps.html" target="_blank">Dataset 1</a> <br> The Triceps Skinfold Thickness Dataset. Package must be loaded and data can be accessed through the R command.</p>
</details>
<br> <!-- Publication 1 -->
<details>
<summary>
<strong>Shaw PA, Deffner V, Keogh R, Tooze JA, Dodd KW, Küchenhoff H, Kipnis V, Freedman LS on behalf of Measurement Error and Misclassification Topic Group (TG4) of the STRATOS Initiative (2018): Epidemiologic analyses with error-prone exposures: review of current practice and recommendations. <em>Annals of epidemiology 28 (11): 821–828.</em> DOI: 10.1016/j.annepidem.2018.09.001</strong> <br><br> <strong>Keywords:</strong> Air pollution, Cohort studies, Measurement error, Misclassification, Nutritional epidemiology, Physical activity
</summary>
<p><br></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S104727971830156X?via%3Dihub" target="_blank">Publication Link</a> <br><br> The publication uses data from a literature survey, conducted by the authors.</p>
</details>
<p><br></p>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp("https:\/\/stratostedp\.github\.io\/");
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>